tests/org.jboss.tools.examples.ui.bot.test/resources/servers/wildfly-14.json

```diff
@@ -0,0 +1,9 @@
+{
+	"org.jboss.ide.eclipse.as.reddeer.server.requirement.ServerRequirement.JBossServer": [
+		{
+			"runtime": "${jbosstools.test.wildfly14.home}",
+			"version": "14",
+			"family": "WILDFLY"
+		}
+	]
+} 
```

Comment:
I think old json file could be removed, right?

Reply:
old wildfly-11.json removed.

Reply:
commit is also signed and pushed to github.